### PR TITLE
111 cart page

### PR DESCRIPTION
**Pull request by CamdenKo**
changed store api db to have fully functional cart page

### CODE REVIEW

File: server/api/cart.js [hunk 1]
```diff
@@ -42,8 +44,37 @@ router.delete('/', (req, res, next) => {
   .catch(next)
 })
 
+router.put('/:packageId', (req, res, next) => {
+  console.log('router.put', req.cart.id, req.params.packageId)
```
Comment (jmlag) [inline server/api/cart.js:48]: delete this

File: server/api/cart.js [hunk 2]
```diff
@@ -42,8 +44,37 @@ router.delete('/', (req, res, next) => {
...
+  CartItem.findOne({
+    where: {
+      cartId: req.cart.id,
+      packageId: req.params.packageId,
+    },
+  })
+  .then(cartItem => {
+    if (cartItem){
+      console.log('quantity', req.body)
```
Comment (jmlag) [inline server/api/cart.js:57]: delete

**Review by CamdenKo — COMMENTED** [review]
Good work everyone :)

**Review by jmlag — COMMENTED** [review]
looks ok

**Pull request merged**